### ISSUE TITLE
When connecting to existing MySQL, prisma2 init wizard ends at the introspect step and does not select prisma tools ( Photon, Lift ) nor generate any boilerplate.

### ISSUE THREAD

**Issue by Dajust** [prisma/prisma2#49]
After successfully introspecting MySQL DB, the init wizard terminates without the select Prisma Tools (Photon, Lift), Language, and Boilerplate steps.

The wizard prints the following message after introspecting:

```
✔ Your all set!

  ──────────────────────────────────────────────────

  We created the following files for you:

  newapp/prisma/project.prisma    The datamodel describes your database schema

  ──────────────────────────────────────────────────

  Run the following commands to start developing

  $ cd newapp
  $ prisma2 dev

  Learn more about using Photon and Lift at
  https://www.prisma.io/docs/...
```

Of course, attempting to run `prisma2 dev` as directed fails with the following message:

```
Error
   Error: Error in lift engine for rpc applyMigration:
     [/root/.cargo/git/checkouts/prisma-query-a8c45647247f5d6d/5a9
5ce6/src/connector/mysql.rs:83] self.pool.state() = State {
       connections: 10,
       idle_connections: 10,
   }
   [migration-engine/connectors/sql-migration-connector/src/lib.rs
:273] format!("CREATE SCHEMA IF NOT EXISTS `{}` DEFAULT CHARACTER 
SET latin1;" , &
           self . schema_name) = "CREATE SCHEMA IF NOT EXISTS `tsh
irtify` DEFAULT CHARACTER SET latin1;"
   [/root/.cargo/git/checkouts/prisma-query-a8c45647247f5d6d/5a95c
e6/src/connector/mysql.rs:83] self.pool.state() = State {
       connections: 10,
       idle_connections: 10,
   }
   [migration-engine/connectors/sql-migration-connector/src/sql_mi
gration_persistence.rs:38] m.make_from(barrel_variant) = "CREATE T
ABLE tshirtify._Migration (revision INTEGER NOT NULL AUTO_INCREMEN
T PRIMARY KEY, name TEXT NOT NULL, datamodel TEXT NOT NULL, status
 TEXT NOT NULL, applied INTEGER NOT NULL, rolled_back INTEGER NOT 
NULL, datamodel_steps TEXT NOT NULL, database_migration TEXT NOT N
ULL, errors TEXT NOT NULL, started_at datetime(3) NOT NULL, finish
ed_at datetime(3));"
   [/root/.cargo/git/checkouts/prisma-query-a8c45647247f5d6d/5a95c
e6/src/connector/mysql.rs:83] self.pool.state() = State {
       connections: 10,
       idle_connections: 10,
   }
```
I hope I'm not missing anything.

**Comment by pantharshit00** [comment]
For the error, check your URL in your datamodel. There is a known bug that in cli is writing the port number in the mysql init flow.

I can also confirm the other part of this issue.

**Comment by pantharshit00** [comment]
Update: CLI is writing the correct URL now

**Comment by schickling** [comment]
Will be solved by https://github.com/prisma/prisma2/issues/278

**Comment by divyenduz** [comment]
This is fixed in `prisma2@2.0.0-alpha.150`, please re-open a new issue if the problem persists for you.
